xwiki-platform-core/xwiki-platform-realtime/xwiki-platform-realtime-wysiwyg/xwiki-platform-realtime-wysiwyg-webjar/src/main/webjar/wysiwygEditor.js

```diff
@@ -525,6 +526,68 @@ define('xwiki-realtime-wysiwygEditor', [
 
           realtimeOptions.onLocal();
           createSaver(info);
+
+          // We need to prevent the diff modal from reloading the editor
+          // because when reloading the realtime editor when editing with
+          // another user, the content will not be loaded from the saved
+          // version, but from the existing realtime session.
```

Comment:
I'm wondering if we need to handle the page title. I know the title is not included in the realtime sync, but it still can cause a merge conflict:

* you enter a realtime session
* you modify the title (without saving)
* someone else edits the page outside the realtime session and changes the page title and saves
* you (from the realtime session) save or the autosave kicks in; in both cases the title is pushed with the content changes
* you get the merge conflict modal, and you chose to reload the editor; it reloads only the content, but not the title
* you save, overwriting the title without being aware of it

It's not critical, but we should be aware of it.

Reply:
Indeed, I'm opening a JIRA issue for it. I added a TODO in the code as well.